### PR TITLE
misc fixes on dashboard-2

### PR DESCRIPTION
**Pull request by shadrak98**
- feat: show deployed by columns in deploys list (fixes https://github.com/frappe/press/issues/1413)
- show created_by column in jobs list for both sites and benches (fixes https://github.com/frappe/press/issues/1413)
- fixes https://github.com/frappe/press/issues/1437

### Screenshots:
1.
image

2.
image

3.
image

### CODE REVIEW

File: press/api/client.py [hunk 1]
```diff
@@ -331,6 +331,9 @@ def is_owned_by_team(doctype, docname, raise_exception=True):
 	if not frappe.local.team():
 		return False
 
+	if frappe.local.system_user():
+		return True
```
Comment (netchampfaris) [inline press/api/client.py]: this makes the function name return a "wrong" value, why is this needed?

**Review by netchampfaris — CHANGES_REQUESTED** [review]

File: press/api/client.py (hunk 1 above)
Comment (shadrak98) [reply press/api/client.py]: Removed it and added impersonate button instead.
